spec/feature/unlock_account_spec.rb

```diff
@@ -0,0 +1,74 @@
+RSpec.feature "Unlock account" do
+  include ActiveJob::TestHelper
```

Comment:
I wonder if some of these might do well in a request test?
I figured a feature test was on teh basis of what the user can see - so Ignores things like what's enqueued here, which is why you're having to bring in the TestHelper, no?

So we'd have some tests for user facing stuff (when i click on this i see title x here?)
and then some reqests tests (given state x, if a request y is sent here... then...)

Reply:
Thanks Huw!  As mentioned on Slack, I've now changed these to request tests, since there's no actual user interface being tested here.

Reply:
I've noticed the changes to the `log_in` methods are now unrelated to this PR, but are good housekeeping anyway since we have a helper method that isn't actually being used anywhere (since the decoupling of registration and login).

Reply:
Ace thanks so much! I'll re-review now